lsdcGui.py

```diff
@@ -3123,10 +3137,11 @@ def processBeamSize(self,beamSizeFlag):
       self.beamsizeComboBox.setCurrentIndex(beamSizeFlag)
 
     def processEnergyChange(self,energyVal):
-      if (energyVal<9000):
-        self.beamsizeComboBox.setEnabled(False)
-      else:
-        self.beamsizeComboBox.setEnabled(True)
+      if daq_utils.beamline != "amx":
+        if (energyVal<9000):
+          self.beamsizeComboBox.setEnabled(False)
+        else:
+          self.beamsizeComboBox.setEnabled(True)
```

Comment:
Consider adding 

`print("It's over 9000!") `

Reply:
this is a conservative change that preserves the previous behavior. I won't change it now, but will consult with FMX to see what feedback users should get when the beam size is no longer changeable.